docker-compose.yml

```
@@ -0,0 +1,29 @@
+version: "3"
+services:
+  api:
+    build:
+      context: ./docker
+      dockerfile: Dockerfile.dev.api
+    container_name: api.timelogger
```

🍠
container_nameは、一般的にハイフン（-）かアンダースコア（_）が多いのでそちらに合わせても良いかなと思いました。

```suggestion
    container_name: api-timelogger
```

参考：
https://docs.docker.jp/compose/compose-file/compose-file-v3.html#container-name

---

ありがとうございます！こちら修正します。

---

いくつかのご指摘まとめて、[a3e4d15](https://github.com/commew/timelogger-api/pull/2/commits/a3e4d155c5ec7af1c9bd7bf9d6da466c716bbf2f) にて対応いたしました！